configs/config_sandbox/esrf16_altmodels_study2/train-2dmodels-unet-efficientnetb0.yaml

```diff
@@ -0,0 +1,19 @@
+train_config:
+  model_id_prefix: 'segmentation-model-unet-efficientnetb0-study2'
```

Comment:
convert to this existing naming convention (mostly underscores) and add 'altmodels': 

EXAMPLE:
```
segmentation-model-esrf16_altmodels_study2_fpn_efficientnetb0
```
change file name to mirror: train-esrf16_altmodels_study2_fpn_efficientnetb0.yaml